hack/jenkins/e2e-runner.sh

```diff
@@ -290,6 +243,15 @@ echo "--------------------------------------------------------------------------
 # return this at the very end.
 EXIT_CODE=0
 
+if [[ -z "${GOPATH:-}" ]]; then
+  echo 'Unset GOPATH' >&2
+  exit 1
+fi
+
+# TODO(fejta): remove this workaround due to not checking out the repository
+go get github.com/sergi/go-diff/...
```

Comment:
It's almost certainly a bad idea to do this. Now we'll be broken anytime they push a breaking change.